src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const PRICE_PER_DAY = 40 * days;
+  const DISCOUNT_FROM_SEVEN_DAYS = 50;
+  const DISCOUNT_FROM_THREE_DAYS = 20;
+  const LONG_PERIOD = 7;
+  const SMALL_PERIOD = 3;
+
+  if (days >= LONG_PERIOD) {
+    return PRICE_PER_DAY - DISCOUNT_FROM_SEVEN_DAYS;
+  }
 
+  if (days >= SMALL_PERIOD) {
+    return PRICE_PER_DAY - DISCOUNT_FROM_THREE_DAYS;
+  } else {
+    return PRICE_PER_DAY;
+  }
```

Comment:
```suggestion
  } 
  
  
    return PRICE_PER_DAY;
  }
```
else is redundant here